identity-service/compose/.env

```diff
@@ -34,6 +36,7 @@ blacklisterPublicKey=0xcccc36bE44D106C6aC14199A2Ed6a29fDa25d5Ae
 relayerPrivateKey=34efbbc0431c7f481cdba15d65bbc9ef47196b9cf38d5c4b30afa2bcf86fafba
 relayerPublicKey=0xaaaa90Fc2bfa70028D6b444BB9754066d9E2703b
 relayerWallets=[{"publicKey": "0xaaaa90Fc2bfa70028D6b444BB9754066d9E2703b", "privateKey": "34efbbc0431c7f481cdba15d65bbc9ef47196b9cf38d5c4b30afa2bcf86fafba"}, {"publicKey": "0xBE718F98a5B5a473186eB6E30888F26E72be0b66", "privateKey": "d3426cd10c4e75207bdc4802c551d21faa89a287546c2c6b3d9a0476f34934d2"}]
+ethFunderAddress=0x1eC723075E67a1a2B6969dC5CfF0C6793cb36D25
```

Comment:
how'd you come up with this?

Reply:
This is a random address (I picked it up from the `delegateOwnerWallet`), I was not sure what to use for the default here otherwise.

Reply:
Sorry, realized this was wrong, fixed 👍